### PR TITLE
Remove scrolling conditional and allow auto-detect board when no board selected

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #2551
Fixes: #2609

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/firmware_flasher.js
Comment (McGiverGim) [inline src/js/tabs/firmware_flasher.js:32]: Does the element `$('div.release_info)` exist at this point?

If yes, usually we create directly the element:
```javascript
const divRealeaseInfoElement = $('div.release_info');
```
and we use it later in the rest of the code.

Comment (haslinghuis) [reply src/js/tabs/firmware_flasher.js:32]: The element is not available at that point, that's why I choose this solution to the solve sonar error.

**Review by McGiverGim — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
Im not sure about removing our current autoscroll here @haslinghuis , for me is a good functionality, maybe we can go to a nonscroll fix in some cases?

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [reply src/js/tabs/firmware_flasher.js]: Found out something weird. Using the $(selector) as a constant was not evaluated at the places we are using them.
I mean when using $('selector') it works not when it's assigned to a constant :confused:

**Comment by haslinghuis** [comment]
@Asizon made scrolling conditional

**Review by blckmn — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
